lib/infobox/node.js

```diff
@@ -39,15 +39,26 @@ define(['sorttable', 'snabbdom', 'd3-interpolate', 'moment', 'helper'],
 
     function showSite(d, config) {
       var site = helper.dictGet(d.nodeinfo, ['system', 'site_code']);
-      var rt = site;
+      var name = site;
+      var link = null;
+
       if (config.siteNames) {
         config.siteNames.forEach(function (t) {
           if (site === t.site) {
-            rt = t.name;
+            name = t.name;
+            link = t.link;
           }
         });
       }
-      return rt;
+
+      return function (el) {
+        var a = document.createElement('a');
```

Comment:
a tag without link -> unable to click, but colored like a link.
Not clear what will happen, because it could also be a filter for this sitecode.

Reply:
What about using a text node in this case?

Reply:
i am for a '_blank' target

Reply:
@genofire done